tools/exporter/exporter.py

```diff
@@ -497,6 +519,28 @@ def parse_command_line_arguments(logger):
 
     return config_filename, export_formats, args.list_export_profiles, loop_enabled
 
+def auto_create_config(logger):
+    """
+    Created new directory in current working directory called 'SafetyCulture Audit Exports'. Default config file placed
+    in directory, with user API Token. User is asked for SafetyCulture username and password in order to generate their
+    API token.
+    :param logger:  the logger
+    """
+    directory = 'SafetyCulture Audit Exports'
+    current_directoy_path = os.getcwd()
```

Comment:
This is generally suspicious as it may not be what you think it is e.g. it depends where you run the script from. Do some research for alternatives in Python e.g. find the location the Python script resides in.

Reply:
Since `exporter.py` looks in the current working directory for config.yaml (if a path is not specified) I figured it makes sense to drop the config in the working directory if it's being auto-created. I thought it would be confusing for users if the config is dropped into the location of the script which `pip install` drops into the Python installation `bin`. Plus, the script would need to update to look near itself, rather than from the users working directory.